.github/actions/check-issues-are-disabled/action.yaml

```diff
@@ -0,0 +1,25 @@
+name: Check Issues Are Disabled
+description: Checks whether the current repository has disabled the issues feature.
+
+outputs:
+  issues-disabled:
+    description: Whether the issues are disabled or not.
+    value: ${{ steps.repository-has-disabled-issues.outputs.issues-disabled }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Scripts
+      shell: pwsh
+      run: |
+        "${{ github.action_path }}" >> $Env:GITHUB_PATH
+        (Resolve-Path "${{ github.action_path }}/../../../Scripts").Path >> $Env:GITHUB_PATH
+  
+    - name: Check Issues Are Disabled In Current Repository
+      shell: pwsh
+      id: repository-has-disabled-issues
+      run: |
+        $issueResponse = $(gh issue list --repo '${{ github.repository }}')
+        $message = "the '${{ github.repository }}' repository has disabled issues"
```

Comment:
We don't have analysis here, but in normal PS it would be detected as not using apostrophes for a string (since there's no interpolation). You can also try `'the ''${{ github.repository }}'' repository has disabled issues'` instead.

Reply:
This line has been removed to have a more general check.